sktime/transformations/series/lag.py

```diff
@@ -248,7 +248,7 @@ def _transform(self, X, y=None):
                 Xt = Xt.loc[X_orig_idx]
             # sub-set to shifted index, if "shifted"
```

Comment:
In the comment, there is still a "shifted".

Reply:
fixed